### DISCUSSION TITLE
How to rotate just caption only

### DISCUSSION THREAD

**Discussion by crcucb** [ImageMagick/ImageMagick#4475] [General]
How can I slightly rotate text for a caption?  I tried to rotate the "test caption 1" but it actually rotates the entire background image with the caption.  

"C:\Program Files\ImageMagick-7.1.0-Q16-HDRI\magick.exe" convert "C:\temp\Guthrie-teams-Background-Smaller-test.jpg"  -font "permanent-marker.regular.ttf"  -pointsize 30 -fill rgb(56,49,119)  -draw "text 580,152 'Fri November 12, 2021'"  -font "permanent-marker.regular.ttf"   -size 240x200    -fill rgb(77,71,135)   -pointsize 0   -gravity Center   -background pink    caption:"test caption 1" **-rotate 10**  -geometry -100+20   -composite  -font "permanent-marker.regular.ttf"   -size 240x200    -fill rgb(77,71,135)    -gravity Center   -background pink   caption:"test caption 2"   -geometry +250+20   -composite  "C:\temp\Guthrie-teams-Background-Smaller-test-edited.jpg"

**Comment by snibgo** [comment]
"-rotate" operates on all images in the current list. To limit the effect to just one image, use parentheses, like this:
```
( caption:"test caption 1" -rotate 10 )
```
There must be a space before and after each ( and ).

**Comment by fmw42** [comment]
You can use -annotate 3x3 "some text" to rotate the text by 3 deg. See https://imagemagick.org/script/command-line-options.php#annotate and https://legacy.imagemagick.org/Usage/text/#annotate
